tests/ui/README.md

```diff
@@ -0,0 +1,91 @@
+UI Tests for Add-Ons Discovery Pane (discopane)
+==================================
+
+Thank you for checking out Mozilla's addons-frontend ui test suite.
+This repository contains Selenium tests used to test the discovery pane.
+
+[![license](https://img.shields.io/badge/license-MPL%202.0-blue.svg)](https://github.com/mozilla/addons-frontend/blob/master/LICENSE)
+
+Getting involved
+----------------
+
+We love working with contributors to fill out the Selenium test coverage for addons-frontend,
+but it does require a few skills.
+By contributing to our test suite you will have an opportunity to learn and/or improve your
+skills with Python, Selenium WebDriver, GitHub, Virtualenv, the Page Object Model, and more.
+
+For some resources for learning about these technologies, take a look at our documentation on
+[running Web QA automated tests][running-tests].
+
+All of [these awesome contributors][contributors] have opened pull requests
+against this repository.
+
+
+Questions are always welcome
+----------------------------
+While we take pains to keep our documentation updated, the best source of information is those
+of us who work on the project.
+Don't be afraid to join us in irc.mozilla.org [#mozwebqa][mozwebqa] to ask questions about our
+Selenium tests.
+We also have a [mailing list][mailing_list] available that you are welcome to join and post to.
+
+How to run the tests locally
+-----------------------------------------
+We maintain a [detailed guide][running-tests] to running our automated tests which can be found on the MDN website.
```

Comment:
This guide is likely to be quite different from these tests. Let's remove this sentence.